Dockerfile

```diff
@@ -3,26 +3,19 @@ FROM poldracklab/fitlins
 # Set user back to root
 USER root
 RUN mkdir /data
-RUN chown -R neuro /work /data
 
 # Install neurodebian/datalad
 ARG DEBIAN_FRONTEND=noninteractive
 RUN apt-get update && apt-get install datalad -yq
 
-USER neuro
-
 RUN git config --global user.name "Neuroscout"
 RUN git config --global  user.email "user@example.edu"
 
 # Copy the current directory contents into the container
 COPY [".", "/src/neuroscout"]
 
-USER root
-RUN chown -R neuro /src
-USER neuro
```

Comment:
I think you need this in order to `pip install -e` below.

Reply:
I actually just kept the user at `root`. It didn't seem to cause any problems for running the `jupyterhub` shim. Any other problems I should be aware of?

Reply:
Oh, okay. Not that I know of.